iml-agent/src/action_plugins/action_plugin.rs

```diff
@@ -45,6 +45,19 @@ pub fn create_registry() -> action_plugins::Actions {
         .add_plugin("unmount_many", lustre::client::unmount_many)
         .add_plugin("ha_resource_start", high_availability::start_resource)
         .add_plugin("ha_resource_stop", high_availability::stop_resource)
+        .add_plugin(
+            "ha_resource_create",
+            high_availability::create_single_resource,
+        )
+        .add_plugin(
+            "ha_resource_create_cloned",
+            high_availability::create_cloned_resource,
+        )
+        .add_plugin("ha_resource_destroy", high_availability::destroy_resource)
+        .add_plugin(
+            "create_cloned_client",
```

Comment:
```suggestion
            "ha_create_cloned_client",
```

So it's clearer to RPC callers.